.github/workflows/actions/module-compatibility-handler/node_modules/.yarn-integrity

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
While reviewing, I asked myself this question:

Do we have to track `node_modules` folders in workflows? I'm not 100% sure of the answer, but we're already doing it here:  
https://github.com/game-ci/docker/tree/main/.github/workflows/actions/report-to-backend

👍

Reply:
We should in fact not do this, even if we are already. It's better to transpile so you can treeshake dependencies.

Reply:
I've converted report-to-backend and module-compatibility-handler to typescript and untracked the node modules. I'm not sure if the configs are set up correctly, I just mimicked what I saw in unity-builder. I also don't know how to set up the precommit hooks with multiple ts projects within a subdirectory of the main repo. If I could get some pointers that would be very helpful.